Parsers/ASimWebSession/Parsers/vimWebSessionSquidProxy.yaml

```diff
@@ -111,7 +118,12 @@ ParserQuery: |
     | extend    
       DstHostname = tostring(DstFQDNparts[0]),
       DstDomain = strcat_array(array_slice(DstFQDNparts,1,-1),"."),
-      DstDomainType = "FQDN"
+      DstDomainType = "FQDN",
+      ASimMatchingIpAddr = case(array_length(ipaddr_has_any_prefix) == 0 ,"",
```

Comment:
1. If the address is not in Src or Dst, the record should not be returned.  ipaddr_has_any_prefix is for Src and Dst, not just anything in the record. For example, it should not trigger if the URL has a parameter with the IP address.
2. Since this becomes post filtering, it should be higher up.

Reply:
You are right. not sure why I commented line 96. now uncommented.